ui/raidboss/data/04-sb/dungeon/bardams_mettle.js

```diff
@@ -0,0 +1,313 @@
+'use strict';
+
+// Bardam's Mettle
+
+[{
+  zoneRegex: {
+    en: /^Bardam's Mettle$/,
+    cn: /^巴儿达木霸道$/,
+    ko: /^바르담 패도$/,
+  },
+  zoneId: ZoneId.BardamsMettle,
+  timelineFile: 'bardams_mettle.txt',
+  timelineTriggers: [
+    {
+      id: 'Bardam\'s Mettle Feathercut',
+      regex: /Feathercut/,
+      beforeSeconds: 4,
+      condition: function(data) {
+        return data.role == 'tank' || data.role == 'healer';
+      },
+      response: Responses.tankBuster(),
+    },
+  ],
+  triggers: [
+    {
+      id: 'Bardam\'s Mettle Rush',
+      netRegex: NetRegexes.tether({ id: '0039' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Extend the tether',
```

Comment:
Maybe reuse `Qitana Hound Tether` here? (although in both cases, I wonder if getOut or some new getAway would be a shorter way to put it?)

Reply:
I think the Qitana notification is better here. An experienced healer wouldn't care, but a novice healer might be pressured if a player just moves out of melee range.